pkg/kv/kvserver/raft_transport.go

```diff
@@ -864,6 +903,16 @@ func (t *RaftTransport) SendAsync(
 		}
 	}
 
+	outgoingMessageHandler, ok := t.getOutgoingMessageHandler(req.FromReplica.StoreID)
+	if ok {
+		outgoingMessageHandler.HandleRaftRequestSent(t.AnnotateCtx(context.Background()), req)
+	} else {
+		log.VEventf(t.AnnotateCtx(context.Background()), 2,
```

Comment:
I would drop the command if there is no handler, just to make it more obvious that nothing will work if the handlers aren't plumbed correctly.

Reply:
Done.